tests/test_item_collection.py

```diff
@@ -7,7 +7,7 @@
 
 class TestItemCollectionAPI(unittest.TestCase):
 
-    expected_keys = ["categories", "html"]
+    expected_keys = ["categories", "html", "refund_policy"]
```

Comment:
These strings should be in single quotes as they are not UI strings. Same below.